tests/lib/rules/no-children-prop.js

```diff
@@ -15,6 +15,7 @@ var RuleTester = require('eslint').RuleTester;
 var parserOptions = {
   ecmaVersion: 6,
   ecmaFeatures: {
+    experimentalObjectRestSpread: true,
```

Comment:
let's not necessarily enable this for every test - can we enable it just for the new tests?